backoffice/package.json

```diff
@@ -27,7 +27,9 @@
     "react-select": "^5.7.0",
     "react-toastify": "^9.0.8",
     "slugify": "^1.6.5",
-    "yup": "^0.32.11"
+    "yup": "^0.32.11",
+    "date-fns": "^2.29.3",
```

Comment:
I saw that project already uses a date utility library as MomentJS.
I think the API format date provided by MomentJS can be used to format dates when you export the file names, instead of installing another date utility library.